src/style.css

```diff
@@ -1 +1,51 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  margin: 0;
+  padding: 0;
+
+  display: flex;
+  justify-content: flex-start;
+
+  max-height: 116px;
```

Comment:
Don't fix container size (if there is no such a requirement) [[checklist]](https://github.com/mate-academy/layout_stars/blob/master/checklist.md#css)